src/engine/server/sv_client.cpp

```diff
@@ -40,6 +40,10 @@ Maryland 20850 USA.
 #include "qcommon/sys.h"
 #include <common/FileSystem.h>
 
+Cvar::Cvar<bool> sv_wwwDownload("sv_wwwDownload", "server does a www dl redirect", Cvar::NONE, "");
+Cvar::Cvar<std::string> sv_wwwBaseURL("sv_wwwBaseURL", "base URL for redirect", Cvar::NONE, "");
+Cvar::Cvar<std::string> sv_wwwFallbackURL("sv_wwwFallbackURL", "URL to send to if an http/ftp fails or is refused client side", Cvar::NONE, "");
```

Comment:
Is FTP really supported?

Reply:
No idea, that was specified in a comment. I didn't rephrased those (no time right now to try to understand that sv_www code), only placed them in the cvar description field.

Reply:
Pretty sure FTP isn't supported. It is only mentioned in a few outdated comments server-side I think.

Reply:
I used wording proposed by slipher , so there's no more mentions of FTP. In any case, I don't think it's a good idea to promote FTP overall, so even if supported by accident, no need to advertise the feature.